scripts/size_report/report_binary_size.ts

```diff
@@ -157,7 +158,8 @@ async function collectBinarySize(pkg: string): Promise<Report[]> {
 async function generateSizeReport(): Promise<BinarySizeRequestBody> {
   const reports: Report[] = [
     ...generateReportForCDNScripts(),
-    ...(await generateReportForNPMPackages())
+    ...(await generateReportForNPMPackages()),
+    ...(await generateReportForBundles())
```

Comment:
How long does this task take to finish? If it adds significant latency, we may want to set up a separate workflow for it.

Reply:
The GHA runs for this pull request: 
- [push](https://github.com/firebase/firebase-js-sdk/runs/4170287745?check_suite_focus=true) (overall: 24m 49s, yarn build: 20m 59s, size test: 1m 47s)
- [pull_request](https://github.com/firebase/firebase-js-sdk/runs/4170288216?check_suite_focus=true) (overall: 19m 54s, yarn build: 16m 36s, size test: 1m 49s)

Sounds to me not a big concern. What do you think?

Reply:
Yeah, looks fine to me too, the bottleneck is the build.